src/js/experimental/media-captions-listbox.ts

```diff
@@ -129,15 +134,15 @@ class MediaCaptionsListbox extends MediaChromeListbox {
 
       // add CC icon for captions
       const type = subs.kind ?? 'subs';
-      if (type === 'captions') {
+      if ((type as string) === 'captions') {
         option.append(createIndicator(this, 'captions-indicator'));
       }
 
       container.append(option);
     }
   }
 
-  #onChange() {
```

Comment:
same here, if this change is to satisfy types let me know. I'll help try to find a way so the code doesn't need to change